exercises/practice/secret-handshake/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub fn commands(n: u8) -> Vec<&'static str> {
```

Comment:
how's everyone feeling about the name `commands` here? It seems to be an awkward fit for the situation of giving a handshake to a friend, since I'm not a general commanding my armies. I am thinking of `actions` instead, because our secret handshake consists of a number of actions.

Reply:
Actions could work well!

Reply:
Thanks for the feedback. I used the JS exercise as a reference, where they use `commands`, but `actions` do make a lot of sense and probably is a better fit.